src/components/avatar.rs

```diff
@@ -137,9 +137,15 @@ impl Avatar {
             >| {
                 item.as_ref()
                     .and_then(|i| i.downcast_ref::<Chat>())
-                    .filter(|chat| chat.is_own_chat())
+                    .filter(|chat| chat.is_own_chat() || chat.is_replies_chat())
```

Comment:
This can be implemented more nicely with [filter_map](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter_map).